packages/ui/src/forum/components/threads/MyThreadsBrowser.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react'
+
+import { useMyThreads } from '@/forum/hooks/useMyThreads'
+
+import { ThreadBrowser } from './ThreadBrowser'
+
+export const MyThreadsBrowser = () => {
+  const [currentPage, setCurrentPage] = useState(1)
+  const threadsPerPage = 2
+  const { threads, pageCount, totalCount, isLoading } = useMyThreads({ page: currentPage, threadsPerPage })
```

Comment:
```suggestion
  const { threads, pageCount, totalCount, isLoading } = useMyThreads({ page: currentPage, threadsPerPage: 2 })
```

I think we can remove that variable.